janus_core/helpers/mlip_calculators.py

```diff
@@ -203,6 +206,27 @@ def choose_calculator(
 
         calculator = AlignnAtomwiseCalculator(path=path, device=device, **kwargs)
 
+    elif architecture == "sevennet":
+        from sevenn.sevennet_calculator import SevenNetCalculator
+
+        __version__ = "0.0.0"
+
+        if model_path is None or model_path.name == "":
+            raise ValueError(
+                "Please specify `model_path` there is no "
+                f"default model for {architecture}"
+            )
+        if isinstance(model_path, Path):
+            model = str(model_path)
+        elif isinstance(model_path, str):
+            model = model_path
+        else:
+            model = None
```

Comment:
What does `model = None` do here? `raise`?

Reply:
yes inside the sevennet